x/distribution/keeper/calculation.go

```diff
@@ -0,0 +1,161 @@
+package keeper
```

Comment:
`calculation.go` seems kinda really broad :P
I'd recommend either splitting up by how these functions will be called by abci (aka `transaction.go`, `end_block.go`) or alternatively by affected object type (aka `delegation.go`, `validator.go`)

Reply:
OK, split into `delegation.go`, `validator.go`.